packages/visx-demo/src/sandboxes/visx-bars/Example.tsx

```diff
@@ -16,9 +15,15 @@ export type BarsProps = {
   width: number;
   height: number;
   events?: boolean;
+  data?: readonly LetterFrequency[];
```

Comment:
when reading #1201 , I interpreted it as refactoring a given example to take in fully arbitrary data, not a specific dataset.

so for `Bars` I think that would require TypeScript generics for data and for users to pass in the relevant x/y accessors, for example

```tsx
type BarsProps<Datum> = {
  ...
  data: Datum[];
  xAccessor: (d: Datum) => string; 
  yAccessor: (d: Datum) => number;
}

// usage
<BarsExample
  data={letterFrequency}
  xAccessor={d => d.letter}
  yAccessor={d => d.frequency}
  {...}
/>
```

The `BarStack` example would need to do similar except it would need to have accessors for x/y/stack, and probably also a TypeScript generic for the possible stack keys.

Reply:
Thanks @williaster this makes more sense. I will work on it this week. I wish I saw it over the weekend